saleor/core/utils/taxes.py

```diff
@@ -104,3 +106,18 @@ def get_taxed_shipping_price(shipping_price, taxes):
     if not charge_taxes_on_shipping():
         taxes = None
     return apply_tax_to_price(taxes, DEFAULT_TAX_RATE_NAME, shipping_price)
+
+
+def price(base: Union[TaxedMoney, TaxedMoneyRange], display_gross=None) -> Money:
```

Comment:
As it is not a type itself but a convertion, wouldn't that be better to add at least a verb?

> get_display_price

Or something like this?